### PR TITLE
[NON-MODULAR] Adds sexes to Ethereals

### PR DESCRIPTION
**Pull request by Mega-Drones**
## About The Pull Request

Allows for Ethereals to chose a sex instead of being forced into non-binary.

## How This Contributes To The Skyrat Roleplay Experience

Ethereals should be able to chose a sex for the personal preference of their players

## Changelog

:cl: MegaDrone
qol: Allows for Ethereals to chose their sex
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
This feels like two PRs crammed into one, please split these into two PRs. One for Ethereal sexes, the other for the plasma fixation virus symptom.

**Comment by Mega-Drones** [comment]
Atomized the PR, I'll put the Plasma fixation balance in a separate one

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -407,6 +407,7 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	null, \
 	new/datum/stack_recipe("blindfold", /obj/item/clothing/glasses/blindfold, 2), \
 	new/datum/stack_recipe("eyepatch wrap", /obj/item/clothing/glasses/eyepatch/wrap, 2), /*SKYRAT EDIT ADDITION*/ \
+	new/datum/stack_recipe("eyepatch", /obj/item/clothing/glasses/eyepatch, 2), /*SKYRAT EDIT ADDITION*/ \
```
Comment (GoldenAlpharex) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:410]: Where does that come from?

File: code/modules/mob/living/carbon/human/species_types/ethereal.dm [hunk 2]
```diff
@@ -18,7 +18,7 @@
 	species_traits = list(DYNCOLORS, AGENDER, NO_UNDERWEAR, HAIR, FACEHAIR, HAS_FLESH, HAS_BONE) // i mean i guess they have blood so they can have wounds too
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
 	species_language_holder = /datum/language_holder/ethereal
-	sexes = FALSE //no fetish content allowed
+	sexes = TRUE //Fetish content enabled
```
Comment (GoldenAlpharex) [inline code/modules/mob/living/carbon/human/species_types/ethereal.dm:21]: This isn't a modular file so you should mark it with a `SKYRAT EDIT - ORIGINAL: FALSE`

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (Mega-Drones) [reply code/game/objects/items/stacks/sheets/sheet_types.dm:410]: Accidently used old branch, gonna remake the PR

**Pull request closed without merging**